node/env0-deploy-cli.js

```diff
@@ -73,17 +75,26 @@ const run = async () => {
   }
 };
 
-const getEnvironmentVariablesOptions = (environmentVariables) => {
+const parseEnvironmentVariables = (environmentVariables, sensitive) => {
   const result = [];
+
   if (environmentVariables && environmentVariables.length > 0) {
-    console.log('getting Environment Variables from options:', environmentVariables);
+    console.log(`getting ${sensitive ? 'Sensitive ' : ''}Environment Variables from options:`, environmentVariables);
```

Comment:
Wait, doesn't this print out the environmentVariables even if they are sensitive?

Reply:
Nevermind, I just figured it doesn't matter at all. The user already puts the sensitive variable in the CLI